### DISCUSSION TITLE
How to serve a static website on a specific route?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [nitrojs/nitro#2069] [General]
How to serve a static website on a specific route?

**Comment by impeterk** [comment]
Hi I have the same question. So here is situation. I want build an full stack application, I know, i can use any fs framework. 
Anyway, so I would like to use nitro as server, right, for all the server things like api, auth, etc... And if I go to index page I want my static html with script for my spa to be served.

**Reply by TheAlexLichter** [reply]
So, uhh, you want https://nuxt.com/ ? 🤔

**Reply by ivanjeremic** [reply]
> So, uhh, you want https://nuxt.com/ ? 🤔

No, we want to stay in nitro but get this.

**Reply by TheAlexLichter** [reply]
Okay. And what keeps you from using Nuxt with that one route and copy all your nitro code in the `server` folder? Give it at try!

**Reply by ivanjeremic** [reply]
> Okay. And what keeps you from using Nuxt with that one route and copy all your nitro code in the `server` folder? Give it at try!

I now know more about Nitro and found a solution, to your question about why not Nuxt, while I respect the work you guys do in all those projects the main reason is that I don't want to use Vue, Nitro for me is the perfect middle-ground for now and I hope that both projects will share more agnostic packages in the future, I try to learn more about the unjs ecosystem and contribute back in the future if I can.

**Reply by impeterk** [reply]
> So, uhh, you want https://nuxt.com/ ? 🤔

Yes, but I want to build it myself... If that make sense, to understand it better maybe, or just for fun.
The good old if you are not using JS framework you are building one.

**Comment by meirlamdan** [comment]
https://nitro.unjs.io/guide/assets#public-assets


```
public/index.html  //     http://localhost:3000/
public/a/index.html  //    http://localhost:3000/a
public/a/b.html  //    http://localhost:3000/a/b.html
